app/src/pages/CardSubmission/WaitingForSubmissions.tsx

```diff
@@ -1,23 +1,23 @@
-import { Button, Grid, Typography } from "@material-ui/core"
+import { Grid, Typography } from "@material-ui/core"
 import Fishbowl from "components/FishbowlAnimation"
 import PlayerChip from "components/PlayerChip"
 import { CurrentGameContext } from "contexts/CurrentGame"
 import { CurrentPlayerContext, PlayerRole } from "contexts/CurrentPlayer"
-import {
-  GameStateEnum,
-  useUpdateAllPlayersMutation,
-  useUpdateGameStateMutation,
-} from "generated/graphql"
-import { teamsWithSequence } from "lib/team"
 import { reject, some } from "lodash"
 import { Title } from "pages/CardSubmission"
 import * as React from "react"
+import AssignTeamsButton from "pages/CardSubmission/AssignTeamsButton"
+import BowlCard from "components/BowlCard"
+
+enum WaitingForSubmissionsState {
+  Waiting,
+  SubmittedAssign,
+  SubmittedWait,
+}
```

Comment:
This effectively represents the previous set of mutually exclusive states, but perhaps we want to change this (i.e. with screening verbiage, for example)?

Reply:
i'll take a pass at this. merging for now into the base branch to review all at once